test/cli/fixtures/imports_peggy.peggy

```diff
@@ -0,0 +1,25 @@
+import "./lib.js" // Just initializers
+import {} from "./lib.js" // Just initializers
+import bar from "./lib.js" // Default rule
+import {baz} from "./lib.js" // One rule imported into this namespace
+import {
+  boo, bab,
```

Comment:
According to Wikipedia the canonical metasyntactic variables are `foo`, `bar`, `baz`, `qux`, `quux`, etc.

Reply:
I don't think this is worth changing in a test.